pkg/virt-api/webhooks/fuzz/fuzz_test.go

```diff
@@ -186,7 +186,7 @@ func fuzzKubeVirtConfig(seed int64) *virtconfig.ClusterConfig {
 				virtconfig.Root,
 				virtconfig.ClusterProfiler,
 				virtconfig.WorkloadEncryptionSEV,
-				virtconfig.DockerSELinuxMCSWorkaround,
+				deprecation.DockerSELinuxMCSWorkaround,
```

Comment:
Here I would say we want to run default config but I see that this needs more thinking... so let's leave it